zetaclient/testutils/mocks/solana_rpc.go

```diff
@@ -47,6 +47,36 @@ func (_m *SolanaRPCClient) GetAccountInfo(ctx context.Context, account solana.Pu
 	return r0, r1
 }
 
+// GetBalance provides a mock function with given fields: ctx, account, commitment
+func (_m *SolanaRPCClient) GetBalance(ctx context.Context, account solana.PublicKey, commitment rpc.CommitmentType) (*rpc.GetBalanceResult, error) {
```

Comment:
Can this `GetBalance` mock be documented, specially the lines where you check `ret.Get(0)` and `ret.Get(1)`, and the reason of doing so. The thing here is on this cases it's really easy to forget what the logic was intended to do.

Reply:
It's generated code.

Reply:
Oops! :)
Then disregard completely, I see know it's generated by our mocks.